atd-vze/package.json

```diff
@@ -67,7 +67,7 @@
     "react-scripts": "^3.4.4"
   },
   "scripts": {
-    "start": "export HTTPS=true && react-scripts start",
+    "start": "export HTTPS=false && react-scripts start",
```

Comment:
I'm disabling SSL here. The previous configuration uses a unsigned (snake-oil) certificate and I am unsure of the value of making developers acknowledge this, when the network traffic in question is not traversing the internet.

Reply:
👍 I remember this being significant but I have no idea why. I found the PR where is was added, and I [reacted with an emoji to that change](https://github.com/cityofaustin/atd-vz-data/pull/110/files#r314878942). None of which is helpful. 😆 The app starts up just fine either way so I'm supporting it since it only applies to local dev anyways.

Reply:
Oh Mike, that is the hint I needed. Peeking at that PR, i see this addition: `REACT_APP_MAPBOX_TOKEN="PRIVATE TOKEN GOES HERE"`. 

I bet mapbox won't serve up background tiles if we mix and match HTTPS & HTTP. Mapbox also won't serve them up without overloading the referrer, so I am not entirely sure there is any benefit to enabling local SSL, unless we all agree that we're going to cohesively fib to mapbox in our request headers. Thanks for this - we know how to turn it back on if we need to, and I'm totally open to conversations about why we should leave it on anyway, if anyone feels strongly on the subject.